modules/api/pkg/machine/convert.go

```diff
@@ -215,6 +215,9 @@ func GetAPIV2NodeCloudSpec(machineSpec clusterv1alpha1.MachineSpec) (*apiv1.Node
 		if config.RootDiskSizeGB != nil && *config.RootDiskSizeGB > 0 {
 			cloudSpec.Openstack.RootDiskSizeGB = config.RootDiskSizeGB
 		}
+		if cd := config.ConfigDrive.Value; cd != nil {
```

Comment:
I meant removing this and just using `omitempty`. Could that work?

Reply:
Here the `config` comes from the machine-controller (reference: https://github.com/kubermatic/machine-controller/blob/a9dc4e656d96ab9a29f4c7e34fbc7ad3c79237d2/sdk/cloudprovider/openstack/types.go#L55).

The `ConfigDrive` value is a Go struct called [`providerconfig.ConfigVarBool`](https://github.com/kubermatic/machine-controller/blob/a9dc4e656d96ab9a29f4c7e34fbc7ad3c79237d2/sdk/providerconfig/types.go#L274) that is used by other fields in machine-controller as well; so, adding omitempty may cause unexpected results in other fields in my opinion.